lib/focuslight/data.rb

```diff
@@ -4,73 +4,72 @@
 require "focuslight/graph"
 require "sequel"
 
-DB = Sequel.connect(Focuslight::Config.get(:dburl), logger: Focuslight.logger)
-
 class Focuslight::Data
   include Focuslight::Logger
 
   def initialize
+    @db = Sequel.connect(Focuslight::Config.get(:dburl), logger: Focuslight.logger, timeout: Focuslight::Config.get(:dbtimeout))
     @datadir = Focuslight::Config.get(:datadir)
     @floatings = Focuslight::Config.get(:float_support) == "y"
 
-    if DB.database_type == :sqlite
-      DB.run 'PRAGMA journal_mode = WAL'
-      DB.run 'PRAGMA synchronous = NORMAL'
+    if @db.database_type == :sqlite
+      @db.run 'PRAGMA journal_mode = WAL'
+      @db.run 'PRAGMA synchronous = NORMAL'
     end
-    @graphs = DB.from(:graphs)
-    @complex_graphs = DB.from(:complex_graphs)
+    @graphs = @db.from(:graphs)
+    @complex_graphs = @db.from(:complex_graphs)
   end
 
   def number_type
-    @floatings ? Float : Bignum
+    @floatings ? Float : Integer
   end
 
   def create_tables
     ntype = number_type
-    DB.transaction do
 
-      DB.create_table :graphs do
-        primary_key :id, Bignum # Notice that SQLite actually creates integer primary key
+    @db.transaction do
```

Comment:
> __[RuboCop]__ Metrics/BlockLength: Block has too many lines. [31/30] ([link](https://sider.review/gh/repos/15607449/pulls/36#issue-3997018))

Close, Because since the db a schema created.